test/test_proxy_tensor.py

```diff
@@ -1413,6 +1413,15 @@ def f(x):
             return r.view(12, -1, 192)
         make_fx(f, tracing_mode="symbolic")(torch.tensor(24))
 
+    def test_view_divisibility_unbacked_relatively_prime(self):
+        # See https://github.com/pytorch/pytorch/issues/123651
+        def f(x):
+            i0 = x.item()
+            torch._check_is_size(i0)
+            torch._check(i0 <= 1000000)
```

Comment:
Why was this changed from 448? Other than that, lgtm

Reply:
Oh oops, I changed this for debugging and forgot to change it back